terraform/workspaces/infra/postgres/migrations.tf

```diff
@@ -0,0 +1,22 @@
+# v2.0.0 -> v2.1.0
+# Added support for multiple Postgres databases. Used for high volume installations
+moved {
+  from = aws_db_instance.postgres
+  to   = aws_db_instance.postgres["paragon"]
+}
+
+moved {
+  from = random_string.postgres_root_username
+  to   = random_string.postgres_root_username["paragon"]
+}
+
+moved {
+  from = random_string.postgres_root_password
+  to   = random_string.postgres_root_password["paragon"]
+}
+
+# incase migrating from single instance to multi, migrate db to zeus so data not erased
+moved {
+  from = aws_db_instance.postgres["paragon"]
+  to   = aws_db_instance.postgres["zeus"]
+}
```

Comment:
It looks like this migration creates a conflict. Using `MULTI_POSTGRES=false` on initial creation yields the following error:

```
│ Error: Moved object still exists
│
│   on postgres/migrations.tf line 19:
│   19: moved {
│
│ This statement declares a move from
│ module.postgres.aws_db_instance.postgres["paragon"], but that resource
│ instance is still declared at postgres/rds.tf:76,14.
│
│ Change your configuration so that this instance will be declared as
│ module.postgres.aws_db_instance.postgres["zeus"] instead.
╵
Operation failed: failed running terraform plan (exit 1)
(node:1) UnhandledPromiseRejectionWarning: Error: Failed command "terraform -chdir=/usr/src/app//terraform/workspaces/infra plan"
```